app/config/featureSwitches/FeatureSwitch.scala

```diff
@@ -22,7 +22,7 @@ sealed trait FeatureSwitch {
 
 object FeatureSwitch {
   val prefix: String = "feature.switch"
-  val listOfAllFeatureSwitches: List[FeatureSwitch] = List(CallETMP,CallPEGA, CallDES, CallAPI1812ETMP, CallAPI1811ETMP, UseAPI1812Model)
```

Comment:
We still want the call feature switches, not the use feature switches (i.e. remove UseAPI1812Model not CallAPI1812ETMP and CallAPI1811ETMP)